packages/cli-e2e/setup.ts

```diff
@@ -23,8 +23,12 @@ async function clearChromeBrowsingData(browser: Browser) {
 
 export default async function () {
   mkdirSync(SCREENSHOTS_PATH, {recursive: true});
-  process.env.GITHUB_ACTION =
-    process.env.GITHUB_ACTION || randomBytes(16).toString('hex');
+  process.env.GITHUB_ACTION = (
+    process.env.GITHUB_ACTION || randomBytes(16).toString('hex')
+  )
+    .replace(/^_*/g, '') // Remove all `-` prefix
```

Comment:
Instead of those comments, you can create the regex with a meaningful name.

```
const leadingDashPrefix = /^_*/g;
const allUnderscore = /_/g;
const allDashFollowedByADigit = /-*(?=\d)/g;

process.env.GITHUB_ACTION = (
    process.env.GITHUB_ACTION || randomBytes(16).toString('hex')
  )
  .replace(leadingDashPrefix, '')
  .replace(allUnderscore, '-')
  .replace(allDashFollowedByADigit, '')
```

And I would instead add a comment that explains why that's important/meaningful ( `Angular needs url friendly.... etc` )


Related to that, if angular really needs URL friendly text, can't we just do  
```
process.env.GITHUB_ACTION = encodeURIComponent(process.env.GITHUB_ACTION || randomBytes(16).toString('hex'))
```

Reply:
(not on PC anymore till monday, so as mentioned in Slack feel free to bring changes if need be)

But:
Sure for the var name, that actually is a good idea 👍 

For the encoding, no 'cause 
![image](https://user-images.githubusercontent.com/12366410/124031046-d4af2e80-d9c4-11eb-9979-7a160fedc9d8.png)
And also because `%20` and stuff like that is not really readable and can hurt the understanding of the logs imho.

Reply:
Looks like the the replace is not doing what the description is saying
```
replace(/^_*/g, '') // Remove all `-` prefix
```
The regex replaces underscores... not dashes